scopes/webpack/config-mutator/config-mutator.ts

```diff
@@ -291,6 +291,33 @@ export class WebpackConfigMutator {
     this.raw = merged;
     return this;
   }
+
+  /**
+   * Add PostCSS plugins
+   * @param plugins
+   * @returns
+   * @example
+   * addPostCssPlugins([require('tailwindcss')]);
+   */
+  addPostCssPlugins(plugins: Array<any>): WebpackConfigMutator {
+    this.raw.module?.rules?.forEach((rule: any) => {
+      if (rule.oneOf) {
```

Comment:
@nachoaldamav 
The config mutator is a generic mutator.
So it might be used for configs that are not our configs.
So sometimes the postcss config is not inside a "one of" rule.
so you need to check it as well